### ISSUE TITLE
Can't delete user account, "yes" option is greyed out

### ISSUE THREAD

**Issue by ValentinBuira** [MerginMaps/mobile#3550]
It's impossible to delete an account because the "yes" option is greyed out despite the name being correct (see attached picture)

How to reproduce : write the account name in the field, and the "yes" button stay greyed out

Current workaround, if you turn back one level in the form, then try again the name of the account while be saved in the field and the "yes" option available

<img src="https://github.com/user-attachments/assets/9110aedc-3b09-4451-a3ca-e7eeb0e77bdf" height="650">

**Comment by tomasMizera** [comment]
Yeah.. the issue is with the "unconfirmed text", if you add a space and then remove the space, it should work. It also behaves differently with different Android keyboards. GBoard should work fine I believe
